.drone.yml

```diff
@@ -0,0 +1,38 @@
+pipeline:
+  style:
+    image: python:3.7-slim
+    environment:
+      PIP_DISABLE_PIP_VERSION_CHECK: 'True'
+    commands:
+    - pip install flake8 'flake8-import-order>=0.18'
+    - flake8
+
+  tests-py3:
+    image: python:3.7-slim
+    environment:
+      SLUGIFY_USES_TEXT_UNIDECODE: 'yes'
+      PIP_DISABLE_PIP_VERSION_CHECK: 'True'
+      PYTEST_ADDOPTS: "--color=yes"
+    commands:
+    - |
+        apt-get update && \
+        apt-get install --no-install-recommends -y \
+          gcc \
+          libc-dev
+    - pip install -e .
+    - python setup.py test
+
+  tests-py2:
```

Comment:
@ashb you could probably run the py2/py3 tests in parallel for a potentially quicker build.